packages/host/app/components/operator-mode/container.gts

```diff
@@ -95,12 +102,19 @@ export default class OperatorModeContainer extends Component<Signature> {
   @tracked searchSheetTrigger: SearchSheetTrigger | null = null;
   @tracked isChatVisible = false;
 
+  @tracked cardForCardMode: CardDef | null = null;
+  @tracked isCodePathInvalid: boolean = false;
+  @tracked codePathErrorMessage: string | null = null;
+
   private deleteModal: DeleteModal | undefined;
 
   constructor(owner: unknown, args: any) {
     super(owner, args);
 
     this.messageService.register();
+    scheduleOnce('afterRender', this, () =>
```

Comment:
why are we scheduling this after render? it's an async task does it matter when it's kicked off?